app/logical/elastic_post_query_builder.rb

```diff
@@ -347,20 +344,21 @@ def build
     when /\A(?<column>#{TagQuery::COUNT_METATAGS.join('|')})(_(?<direction>asc))?\z/i
       column = Regexp.last_match[:column]
       direction = Regexp.last_match[:direction] || "desc"
-      order.concat([{column => direction}, {id: direction}])
+      order.push({column => direction}, {id: direction})
 
     when "tagcount"
       order.push({tag_count: :desc})
 
     when "tagcount_asc"
       order.push({tag_count: :asc})
 
-    when /(#{TagCategory::SHORT_NAME_REGEX})tags/
```

Comment:
This was the bug; b/c it wasn't anchored & it preceded the case for `_asc`, it ate those inputs